engine/src/main/java/org/pentaho/di/job/Job.java

```diff
@@ -697,7 +698,7 @@ private Result execute( final int nr, Result prev_result, final JobEntryCopy job
       ( (VariableSpace) cloneJei ).copyVariablesFrom( this );
       cloneJei.setRepository( rep );
       if ( rep != null ) {
-        cloneJei.setMetaStore( rep.getMetaStore() );
```

Comment:
why does this need to be in an if block that checks rep?

Reply:
I wasn't really sure what this was doing so for all instances that were calling getRepositoryMetastore I converted it to get default unless it was directly responsible for setting up the repository metastore instance